pkg/yurtadm/README.md

```diff
@@ -0,0 +1,63 @@
+# Yurtadm
+
+## 1.Background
+In order to allow users to quickly obtain an OpenYurt test cluster, OpenYurt provides the command `yurtadm init` to initialize the cluster. Users only need to select the version of the OpenYurt cluster mirror to create the corresponding version of OpenYurt. Then yurt-app-manager, yurt-controller-manager, yurt-tunnel-server, yurt-tunnel-agent components will be automatically deployed.
+To expand the cluster later, users can use the `yurtadm join` command to add edge nodes or cloud nodes to the cluster.
+
+## 2.Ability
+Using yurtadm, you can do:
+- Create a simple openyurt cluster with just one command.
+- Create a High-Availability OpenYurt cluster.
+
+## 3.Process
+### 3.1 Compile Yurtadm
+When initializing the cluster, you need to obtain the Yurtadm executable first. To quickly build and install yurtadm, you can execute the following command to complete the installation if the build system has golang 1.13+ and bash installed:
+
+```bash
+git clone https://github.com/openyurtio/openyurt.git
+cd openyurt
+make build WHAT="yurtadm" ARCH="amd64" REGION=cn
+```
+
+The executable will be stored in the _output/ directory.
+
+### 3.2 Initialize the OpenYurt cluster
+Execute the following command as root account, no need to install container runtimes such as docker in advance. Docker will be automatically installed during the execution of `yurtadm init`.
+
+```bash
+# Initialize an OpenYurt cluster.
+yurtadm init --apiserver-advertise-address 192.168.152.131 --openyurt-version latest --passwd 1234
```

Comment:
what's the mean of the address `192.168.152.131` and passwd `1234`? where do end users can find these kind of information?

Reply:
It is just an example. This is the IP address and SSH password of the node the user needs to init. Users can use `yurtadm init -h`  to see the effect of these parameters.
![image](https://user-images.githubusercontent.com/43909149/180733204-b28c39ed-242d-460f-afda-46baca7ed04c.png)

Reply:
i think we should make clear introduction in the tutorial for concrete ip address and password, if not, end user will not understand the meaning.